scripts/Makefile

```diff
@@ -0,0 +1,23 @@
+srcdir = $(CURDIR)/..
+incdir = $(srcdir)/include
+
+CC  := gcc
+CXX := g++
+CFLAGS   := -fPIC -shared -g -I$(incdir)
+CXXFLAGS := $(CFLAGS)
+
+CC	= gcc
+SRCS	= simple.c info.c count.c dump.c simple.cpp
+CSO     = $(SRCS:%.c=%_c.so)
+CPPSO   = $(SRCS:%.cpp=%_cpp.so)
+
+all: $(CSO) $(CPPSO)
```

Comment:
This needs to be called from the top level Makefile.  But I think you should split the addition of examples to a separate commit.